### ISSUE TITLE
Shulker Boxes allowed in shulk origin

### ISSUE THREAD

**Issue by Kelbbie** [LemonyPancakes/Origins-Bukkit#45]
### Minecraft Server's Version and Software

Petal 1.19.1-pre-1

(https://github.com/Bloom-host/Petal)

### Plugin's Version

Latest

### Bug Description

Shulker boxes are allowed to be put in the shulk class's inventory

### To Reproduce

Put a shulker box in the shulk's inventory by pressing F

### Additional Information

This should at least be able to be disabled as having a capacity of 15,000 items in a server setting is a bit too overpowered.

https://i.imgur.com/Ppxmoy6.png

**Comment by LemonyPancakes** [comment]
If I am not mistaken, in the fabric version, shulker boxes are allowed in your shulker inventory.

**Comment by Kelbbie** [comment]
yeah, you are correct, it would be nice if it could be toggled to be disabled. Realistically, I was more so concerned about if a dupe bug was discovered, since this is spigot and ran in a server setting, it could lead to shulkers being hidden in inventories that cant be checked. Is there a way to check the inventories?

**Comment by LemonyPancakes** [comment]
I'll find a way.

**Comment by Kelbbie** [comment]
@LemonyPancakes is this resource depreciated now? i msg'd you on bukkit 2 months ago and you made commits last week
